feature-libs/cart/base/styles/components/_item-counter.scss

```diff
@@ -79,6 +80,14 @@
 
     &:disabled {
       color: var(--cx-color-light);
+      cursor: not-allowed;
```

Comment:
Let's leave the styles without a feature flag unchanged.

Reply:
I've updated it.